docs/content/docs/configuration/cli.md

```diff
@@ -14,9 +14,7 @@ weight: 1
 {{< /cards >}}
 
 > [!NOTE]
-> The formatters explicitly defined in the configuration section [`formatters`](/docs/configuration/file/#formatters-configuration) are automatically run by the command [`golangci-lint run`](/docs/configuration/cli/#run).
-> 
-> You don't need to run [`golangci-lint fmt`](/docs/configuration/cli/#fmt) before [`golangci-lint run`](/docs/configuration/cli/#run)
+> The formatters explicitly defined in the configuration section [`formatters`](/docs/configuration/file/#formatters-configuration) are automatically run (as linter) by the command [`golangci-lint run`](/docs/configuration/cli/#run).
```

Comment:
Can we mention something around the difference between checking if the code is formatted vs. actually formatting it? Thinking similar to `ruff check` or `prettier --check`?

Maybe not perfect but you get the gist:

```suggestion
The [`golangci-lint run`](/docs/configuration/cli/#run) command executes all enabled linters and the formatters defined in [`formatters`](/docs/configuration/file/#formatters-configuration) to verify code formatting.  
> To reformat code, use `golangci-lint fmt`. To apply both formatting and linter fixes, use `golangci-lint run --fix`.
```